extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs

```diff
@@ -160,6 +160,11 @@ private bool IsBlobExtension(ModelBindingData bindingData)
                 throw new ArgumentNullException(nameof(containerName));
             }
 
+            if (targetType != typeof(BlobContainerClient) && string.IsNullOrEmpty(blobName))
```

Comment:
@liliankasem Highlighting that this check may change when we remove `CollectionModelBindingData`. Below code is valid and it does not have target type as `BlobContainerClient`

`BlobInput("container-path") IEnumerable<string> blobs`

Reply:
Ah yup I have that covered in my [PR](https://github.com/Azure/azure-functions-dotnet-worker/pull/1690)